addons/website/static/src/systray_items/website_switcher.js

```diff
@@ -11,19 +12,48 @@ const { Component } = owl;
 export class WebsiteSwitcherSystray extends Component {
     setup() {
         this.websiteService = useService('website');
+        this.notificationService = useService("notification");
+        this.actionService = useService("action");
+        this.noDomainTooltip = this.env._t("This website does not have a domain set.");
     }
 
     getElements() {
         return this.websiteService.websites.map((website) => ({
             name: website.name,
             id: website.id,
+            noDomain: !website.domain,
             callback: () => {
                 if (website.domain && !wUtils.isHTTPSorNakedDomainRedirection(website.domain, window.location.origin)) {
                     const { location: { pathname, search, hash } } = this.websiteService.contentWindow;
                     const path = pathname + search + hash;
                     window.location.href = `${website.domain}/web#action=website.website_preview&path=${encodeURIComponent(path)}&website_id=${encodeURIComponent(website.id)}`;
                 } else {
                     this.websiteService.goToWebsite({ websiteId: website.id });
+                    if (!website.domain) {
+                        const closeFn = this.notificationService.add(
+                            this.env._t(
+                                "This website does not have a domain configured. To avoid unexpected behaviours during website edition, we recommend closing other tabs.\nTo remove this message please set a domain in your website settings"
```

Comment:
```suggestion
                                "This website does not have a domain configured. To avoid unexpected behaviours during website edition, we recommend closing other browser tabs.\nTo remove this message please set a domain in your website settings"
```

Not sure `tabs` really is meaningful without `browser` next to it, it's really technical.

I would also maybe add this:
```suggestion
                                "This website does not have a domain configured. To avoid unexpected behaviours during website edition, we recommend closing (or refreshing) other browser tabs.\nTo remove this message please set a domain in your website settings"
```